chainer/initializations.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+# Original code forked from MIT licensed keras project
+# https://github.com/fchollet/keras/blob/master/keras/initializations.py
+
+
+def init_weight(weights, initWith, scale=1,
```

Comment:
I realized `init_weight` can handle not only matrices but also tensors with arbitrary dimension as weights. Actually, existing `initializer`s like `uniform` or `normal` do be able to initialize tensors. So could you change the description of `init_weight` to support tensors?